src/www/ui/api/documentation/openapi.yaml

```
@@ -3273,6 +3329,49 @@ components:
           description: Date from which to remove older log files from repository.
           nullable: true
           example: "2022-07-16"
+    ScheduleBulkScan:
+      type: object
+      properties:
+        bulkActions:
+          type: array
+          items:
+            type: object
+            properties:
+              licenseId:
+                type: integer
+                example: 1
+              reportinfo:
```

```suggestion
              licenseText:
```
Also, please add descriptions to the properties (which I believe you did with example).